src/main/java/frc/robot/utilities/DebugEntry.java

```diff
@@ -1,80 +1,101 @@
-      localConsumer = (a) -> datalog.appendDouble(localEntry, (Double) a, 0);
+      int localEntry = datalog.start(uri, "double");
+      logConsumer = (a) -> datalog.appendDouble(localEntry, (Double) a, 0);
+
+      DoublePublisher publisher = instance.getDoubleTopic(uri).publish();
+      sendToNT =
+          (data) -> {
+            publisher.accept((Double) data);
+          };
 
       // if String
     } else if (defaultValue instanceof String) {
-      localEntry = datalog.start("/" + subsystem.getName() + "/" + name, "string");
-      localConsumer = (a) -> datalog.appendString(localEntry, (String) a, 0);
+      int localEntry = datalog.start(uri, "string");
+      logConsumer = (a) -> datalog.appendString(localEntry, (String) a, 0);
 
+      StringPublisher publisher = instance.getStringTopic(uri).publish();
+      sendToNT = (data) -> publisher.accept((String) data);
       // if Boolean
     } else if (defaultValue instanceof Boolean) {
-      localEntry = datalog.start("/" + subsystem.getName() + "/" + name, "boolean");
-      localConsumer = (a) -> datalog.appendBoolean(localEntry, (Boolean) a, 0);
+      int localEntry = datalog.start(uri, "boolean");
+      logConsumer = (a) -> datalog.appendBoolean(localEntry, (Boolean) a, 0);
+
+      BooleanPublisher publisher = instance.getBooleanTopic(uri).publish();
+      sendToNT = (data) -> publisher.accept((Boolean) data);
       // if Other
     } else {
-      DriverStation.reportWarning("Unsupported data type.", false);
-    }
+      int localEntry = datalog.start(uri, "string");
+      logConsumer = (a) -> datalog.appendString(localEntry, (String) a.toString(), 0);
 
-    if (!Robot.isCompetition && localEntry != null) {
-      networkTab = Shuffleboard.getTab(subsystem.getName());
+      StringPublisher publisher = instance.getStringTopic(uri).publish();
+      sendToNT = (data) -> publisher.accept(data.toString());
+    }
 
-      if (!entries.containsKey(name)) {
-        networkEntry = networkTab.add(name, defaultValue);
-        entries.put(name, networkEntry);
-      } else {
-        String errorMessage =
-            "Duplicate ShuffleboardEntry on " + networkTab.getTitle() + " tab: " + name;
-        DriverStation.reportError(errorMessage, false);
-      }
+    if (!Robot.isCompetition) {
+      localConsumer =
+          (a) -> {
+            sendToNT.accept(a);
+            logConsumer.accept(a);
+          };
+    } else {
+      localConsumer = (a) -> logConsumer.accept(a);
     }
-    if (localEntry != null) localConsumer.accept(defaultValue);
+
+    localConsumer.accept(defaultValue);
   }
 
   public DebugEntry<T> withPosition(int colIndex, int rowIndex) {
-    if (!Robot.isCompetition) networkEntry.withPosition(colIndex, rowIndex);
     return this;
   }
 
   public DebugEntry<T> withSize(int width, int height) {
-    if (!Robot.isCompetition) networkEntry.withSize(width, height);
     return this;
   }
```

Comment:
Did these cause some issue?

Reply:
looks like the networkEntry var got removed for some reason. Most likely the reason it was removed